pom.xml

```diff
@@ -56,6 +62,12 @@
       <artifactId>marc4j</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.16.1</version>
```

Comment:
Can the latest 3.19.0 version been used?

Reply:
Yep. Works.

Reply:
Yep. Works.